spec/models/user_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  context '#create' do
+    it 'is valid with the existing name' do
+      expect(User.create(name: 'Lily')).to be_valid
+    end
+
+    it 'is not valid with a blank name' do
+      expect(User.create(name: nil)).to_not be_valid
```

Comment:
- [ ] I figured out where the error is, which is making you see the need for migrations. In your `bin` directory, kindly change the executable file name to `ruby` from `ruby.exe` if you want to run the project in a Unix operating system, else, you can run it on Windows.

### Directory with the executable files:
![image](https://github.com/SaveryIV/blog-app-on-Rails/assets/50990925/61bb34e1-81f6-4947-bfc6-50695bce413f)

- [ ] After successfully running your code, I saw a lot of test errors that you need to fix. Kindly first make sure you can run `rspec`, then fix those errors :

### Failed error tests:
![image](https://github.com/SaveryIV/blog-app-on-Rails/assets/50990925/deccc8d0-ac4b-42f7-882f-01b930e69156)

Reply:
@Shedrack-Sunday Thank you a lot! I'm gonna try to fix it and i update you with new information

Reply:
@Shedrack-Sunday I cant find the ruby file
![Uploading image.png…]()

Reply:
![image](https://github.com/SaveryIV/blog-app-on-Rails/assets/85071975/2d88ce09-ddf9-4225-a539-04328abc58d6)

Reply:
@Shedrack-Sunday pls help me

Reply:
Hello @SaveryIV , please call me on zoom or slack if you're still confused